### ISSUE TITLE
Windows Terminal Tab Title not set to Path since upgrading from V2 to V3

### ISSUE THREAD

**Issue by KiLLeRRaT** [JanDeDobbeleer/oh-my-posh#911]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

The current path used to be displayed in the Title bar in Windows Terminal.  I'm not 100% sure, but I'm pretty sure that it's due to me upgrading from OMP V2 to V3.

Before:
![image](https://user-images.githubusercontent.com/7088084/130391595-4d75524b-9f24-4a15-8ea5-ce6d20b9bf30.png)

After:
![image](https://user-images.githubusercontent.com/7088084/130391634-c64b6471-eb29-46a1-952e-a0658391e6cc.png)

### Version

3.175.0

### Theme

Paradox

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

**Comment by lnu** [comment]
The console title is not set by default with Paradox. You can [customize ](https://ohmyposh.dev/docs/windows#customize)the theme by exporting it and updating [the console title properties](https://ohmyposh.dev/docs/configure#console-title-style).  

example of paradox with custom console title template:
```
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "blocks": [
    {
      "type": "prompt",
      "alignment": "left",
      "segments": [
        {
          "type": "root",
          "style": "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground": "#100e23",
          "background": "#ffe9aa"
        },
        {
          "type": "session",
          "style": "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground": "#100e23",
          "background": "#ffffff"
        },
        {
          "type": "path",
          "style": "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground": "#100e23",
          "background": "#91ddff",
          "properties": {
            "folder_icon": "\uF115",
            "folder_separator_icon": " \uE0B1 ",
            "style": "full"
          }
        },
        {
          "type": "git",
          "style": "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground": "#193549",
          "background": "#95ffa4"
        },
        {
          "type": "python",
          "style": "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground": "#100e23",
          "background": "#906cff",
          "properties": {
            "prefix": " \uE235 "
          }
        },
        {
          "type": "exit",
          "style": "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground": "#ffffff",
          "background": "#ff8080",
          "properties": {
            "prefix": " \uE20F"
          }
        }
      ]
    },
    {
      "type": "prompt",
      "alignment": "left",
      "newline": true,
      "segments": [
        {
          "type": "text",
          "style": "plain",
          "foreground": "#007ACC",
          "properties": {
            "prefix": "",
            "text": "\u276F"
          }
        }
      ]
    }
  ],
  "final_space": false,
  "console_title": true,
  "console_title_template": "{{ .Shell }} in {{ .Folder }}",
  "console_title_style": "template"
}
```

**Comment by KiLLeRRaT** [comment]
Hi @JanDeDobbeleer thanks for getting back.  That is great.  I've updaetd my theme, and it works a treat for `pwsh.exe`.

I did notive it not working for normal `powershell.exe`, and `cmd.exe`

Is there another trick to that?

If, I can definitely live with it because for probably 85% of the time I'm in `pwsh.exe`

Cheers!

![image](https://user-images.githubusercontent.com/7088084/130500340-d28b4279-1219-4f88-8833-c9af41f9a032.png)

**Comment by JanDeDobbeleer** [comment]
@KiLLeRRaT with not working do you mean the title? That should work everywhere as it's a terminal feature, not a shell one. About prompt support, Powershell 5 is supported (though the profile is at a different location) and for CMD there's a possibility with clink, but not officially documented.

**Comment by KiLLeRRaT** [comment]
Righto, gotcha! I used to have omp installed in PowerShell 5 also.  I will configure that separately.  As for CMD, it's weird that it doesn't display the path in the titlebar, but that isn't omp's fault!

Thanks for helping out and the speedy reply.

ps. I can't believe I did not find/see the `console_xxxx` config in the documentation.  I've now found it and read it.  Sorry to have wasted a bit of time!

Cheers and have a good one!
